ws_esp32_s3_matrix.json

```diff
@@ -0,0 +1,48 @@
+{
+  "build": {
+    "arduino":{
+      "ldscript": "esp32s3_out.ld",
+      "partitions": "default.csv",
+      "memory_type": "qio_qspi"
+    },
+    "core": "esp32",
+    "extra_flags": [
+      "-DARDUINO_WS_ESP32_S3_MATRIX",
+      "-DBOARD_HAS_PSRAM",
+      "-DBOARD_FLASH_MODE=QIO",
+      "-DBOARD_PSRAM_TYPE=QIO",
+      "-DARDUINO_USB_MODE=1",
+      "-DARDUINO_USB_CDC_ON_BOOT=1",
+      "-DARDUINO_RUNNING_CORE=1",
+      "-DARDUINO_EVENT_RUNNING_CORE=1"
+    ],
+    "f_cpu": "240000000L",
+    "f_flash": "80000000L",
+    "flash_mode": "qio",
+    "psram_type": "qio",
```

Comment:
remove psram_type

Reply:
Clarification please.

This board uses the ESP32-S3FH4R2 SOC which has both 4MB embedded Quad SPI flash and 2MB embedded Quad SPI PSRAM.

The `flash_mode` and `param_type` above need to be different then?

Reply:
See this one https://github.com/pioarduino/platform-espressif32/blob/develop/boards/aventen_s3_sync.json

There is just the obsolete "ldscript" wrong and should not be there.